net-news/liferea/liferea-1.12.5.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+GNOME2_EAUTORECONF="yes"
```

Comment:
Why do you need to reconfigure it?

Reply:
Thanks, it's not needed anymore.
It was needed in 1.12.2 and i forgot to remove it with next vbump.